watch.go

```diff
@@ -286,6 +281,50 @@ func WatchForChanges(
 	}
 }
 
+// collectPages collects all pages of a single filterList invocation
+// and keeps things happy left
```

Comment:
minor: this maybe a useful function to have in our azblob package.

Reply:
Oh yeah, I'll look at refactoring a bit in follow up tickets. I'm sorting out the local directory support and the polution of the interfaces with *azblob.Option in [User Story 9780](https://dev.azure.com/jitsuin/avid/_workitems/edit/9780): veracity log consistency check command

I can pick this up as a drive by. remind me if you don't see this addressed in the PR for that ticket. thanks